core/trino-main/src/test/java/io/trino/server/ui/TestWebUi.java

```diff
@@ -1300,11 +1302,12 @@ private static String issueToken(Claims claims)
 
         private static Claims createClaims()
         {
-            return new DefaultClaims()
-                    .setIssuer(TOKEN_ISSUER)
-                    .setAudience(OAUTH_CLIENT_ID)
-                    .setSubject("test-user")
-                    .setExpiration(Date.from(Instant.now().plus(Duration.ofMinutes(5))));
+            ClaimsBuilder builder = Jwts.claims();
```

Comment:
The new interface seems to support chaining, so should be able to do
```java
return Jwts.claims()
        ...
        .build();

Reply:
Except for audience setting :(